substrate/frame/message-queue/src/mock.rs

```diff
@@ -367,6 +367,6 @@ pub fn num_overweight_enqueued_events() -> u32 {
 		.count() as u32
 }
 
-pub fn fp(pages: u32, count: u64, size: u64) -> QueueFootprint {
-	QueueFootprint { storage: Footprint { count, size }, pages }
+pub fn fp(ready_pages: u32, pages: u32, count: u64, size: u64) -> QueueFootprint {
```

Comment:
```suggestion
pub fn fp(pages: u32, ready_pages: u32, count: u64, size: u64) -> QueueFootprint {
```
Small one, but maintaining the order of `pages` before `ready_pages` might be better here for people looking in the tests for examples of how they both change in certain cases. With a swapped order people might naively confuse the two when they just see the `fp` function call.